packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -237,25 +237,25 @@ export const Stackbox = () => {
         </Button>
       </div>
       <TokenPicker
-        closeAction={closeModal}
+        closeAction={() => closeModal(ModalId.TOKEN_PICKER)}
         initialFocusRef={searchTokenBarRef}
-        isOpen={openModalId === ModalId.TOKEN_PICKER}
+        isOpen={isModalOpen(ModalId.TOKEN_PICKER)}
         onTokenSelect={selectToken}
       />
       <ConfirmStackModal
-        isOpen={openModalId === ModalId.CONFIRM_STACK}
-        closeAction={closeModal}
+        isOpen={isModalOpen(ModalId.CONFIRM_STACK)}
+        closeAction={() => closeModal(ModalId.CONFIRM_STACK)}
       />
       <Toast
```

Comment:
Thinking out loud and not sure we should even have this toast here now that I think about it. Unless we just want to lay some sort of groundwork when we decide to add it to the flow.

Reply:
Had the same concern, but now that we have it we can leave it as it is. But I prefer in the future if we don't leave unused/mock components throughout the layout.

Reply:
makes sense, I think I can remove it.